atoma-bin/atoma_node.rs

```diff
@@ -225,16 +226,17 @@ async fn main() -> Result<()> {
         register_on_proxy(&config.proxy, *node_small_id, &keystore, args.address_index).await?;
     }
 
-    let (compute_shared_secret_sender, compute_shared_secret_receiver) =
+    let (compute_shared_secret_sender, _compute_shared_secret_receiver) =
         tokio::sync::mpsc::unbounded_channel();
 
+    #[cfg(feature = "tdx")]
```

Comment:
if we want to hide this behind a feature flag, maybe we can put all TDX stuff behind a feature flag, that will reduce the size/compilation time when building without it. But I'm not sure how the code acts when it doesn't have confidentiality support but it receives confidential compute request anyway, what happens than? Maybe we can put everything behind feature flag and just don't expose the endpoint at all. Any idea what's better?